srcpkgs/megapixels/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'megapixels'
+pkgname=megapixels
+version=1.4.3
+revision=1
+build_style=meson
+hostmakedepends="pkg-config glib-devel"
+makedepends="gtk4-devel libzbar-devel"
+short_desc="GTK4 camera app for ARM devices"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/postmarketOS/megapixels"
+distfiles="${homepage}/-/archive/${version}/megapixels-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://gitlab.com/postmarketOS/megapixels/-/archive/${version}/megapixels-${version}.tar.gz"
```

Reply:
Why? This makes the line longer and the url is already full of variables anyway.

Reply:
convention is to avoid it for distfiles ( to have better grep support i think)